project/build/dev/d8/themes/custom/patternlab/source/_meta/_00-head.twig

```diff
@@ -15,6 +15,7 @@
   <link rel="stylesheet" href="../../../../dest/style.css">
   <!-- endinject -->
   <link rel="stylesheet" href="../../../../source/styleguide/custom-styleguide-specific.css">
-
+  <script src="https://use.typekit.net/jhz5ozy.js"></script>
```

Comment:
It's has to do with the twig file it renders three font-weights even regardless if only one weight was brought into the site, what I mentioned is that I brought those font-weights over so our patternlab page would look good rendering, or it would be three renders of the same font-weight via  the twig file